# print(1/0)# giving error
try:
    print(1/0)
except:
    print("There is something wrong please check the error in the code and try again brother")
finally:
    print("It will run no matter what will happens in the code")